### DISCUSSION TITLE
Is this curriculum going into maintenance mode?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/2nd-semester-introduction-to-computer-science#428] [General]
I have heard that this curriculum is going into maintenance mode, meaning that updates will be few. I wanted to ask if that is still true? I have looked at the presumptive replacement; Makecode Arcade. It doesn't support Python. Yes, yes, it has a Typescript to Python translator. That translator fails plenty and doesn't implement Python 3 well. 

Currently, I am teaching Makecode Arcade with blocks then switching to this curriculum with REPL.it. 

1. Some of the students take this class to learn Python. Teaching them something else is bait and switch.
2. Teaching them Python 3 gives them a boost when they learn Python in college or look for an intern job.
3. The curriculum looks like a rehash of the block curriculum. In Python-ish it becomes a series of library calls. That works with blocks, it doesn't teach them much Python. 

I would say there is still a lot of life left in this curriculum.

**Comment by AimeeMRosato** [comment]
Hi @jdonwells! This curriculum is in maintenance mode, meaning that we aren't making any major updates to the lessons or content. This is why there was a delay with answering your discussion topic! :)

I wanted to share a new Microsoft offering that TEALS just started supporting: VS Code for Education's Intro to Python: https://vscodeedu.com/courses/intro-to-python. If you are looking for other python resources, check it out! VS Code for Education just launched if you are looking for another code editor option.
